libs/components/forms/src/lib/modules/form-error/form-errors.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SkyFormErrorsComponent } from './form-errors.component';
+
+describe('SkyFormErrorsComponent', () => {
+  let fixture: ComponentFixture<SkyFormErrorsComponent>;
+  let component: SkyFormErrorsComponent;
+
+  beforeEach(async () => {
+    fixture = TestBed.createComponent(SkyFormErrorsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+    await fixture.whenStable();
+  });
+
+  it('should be toggle announcements', async () => {
+    expect(component).toBeTruthy();
+    expect(fixture.nativeElement.outerHTML).toContain('aria-live="polite"');
+    expect(fixture.nativeElement.outerHTML).toContain('aria-relevant="all"');
+    component.announceErrors = false;
+    fixture.detectChanges();
```

Comment:
another nit: can we add spaces so the different steps in the unit test are more separated, it's a bit hard to follow rn

Reply:
Good call. Updated.